pybossa/api/gold_task_run.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf8 -*-
+# This file is part of PyBossa.
+#
+# Copyright (C) 2018 SciFabric LTD.
+#
+# PyBossa is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# PyBossa is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with PyBossa.  If not, see <http://www.gnu.org/licenses/>.
+"""
+PyBossa api module for exposing domain object TaskRun for gold task runs via an API.
+
+This package adds GET method for:
+    * goldtaskrun
+
+"""
+from flask import request, Response
+from flask.ext.login import current_user
+from api_base import APIBase
+from pybossa.model.task_run import TaskRun
+from pybossa.error import ErrorStatus
+from pybossa.core import task_repo
+from werkzeug.exceptions import MethodNotAllowed, Unauthorized
+from pybossa.util import jsonpify, crossdomain
+from pybossa.core import ratelimits
+from pybossa.ratelimit import ratelimit
+from pybossa.cache.projects import get_project_data
+
+cors_headers = ['Content-Type', 'Authorization']
+error = ErrorStatus()
+
+# cloned fromCompletedTaskRunAPI
```

Comment:
Both completed tasks and this filter task runs based on task properties. Ideally we would make the task_run endpoint more powerful. Is that possible? If not, should we have a single api endpoint that handles both?